lib/shared/api/crypto_data/model/crypto_data_response.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+
+
+part 'crypto_data_response.g.dart';
+
+@JsonSerializable()
+class CryptoDataResponse {
+  final String id;
+  final String symbol;
+  final String name;
+  final String image;
+  // ignore: non_constant_identifier_names
```

Comment:
se for realmente necessário utiliar o ignore, utiliza `// ignore_for_file: non_constant_identifier_names` no topo da page, que ai não precisa ficar repetindo. Mas só utiliza esses ignore se for estritamente necessáiro.

Reply:
Obrigado pela dica!